tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bottom_bar.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+class BottomBar(Base):
+    _remove_locator = (By.ID, 'remove')
+    _done_locator = (By.ID, 'done')
+
```

Comment:
This line should do the trick:
`self.marionette.find_element(self._bottom_bar_locator).rect['height']`

http://marionette-client.readthedocs.org/en/latest/reference.html?highlight=html%20element#marionette_driver.marionette.HTMLElement.rect